src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    subtract(value) {
+      this.result -= value;
+
+      return this.result;
```

Comment:
You can omit the `function` keyword when defining methods in an object to make the code more concise. Use the method shorthand syntax.